docs/sources/articles/dockerfile_best-practices.md

```diff
@@ -55,6 +55,13 @@ container. Decoupling applications into multiple containers makes it much
 easier to scale horizontally and reuse containers. If that service depends on
 another service, make use of [container linking](https://docs.docker.com/userguide/dockerlinks/).
 
+In some cases the process you start might create more processes.
```

Comment:
There are some slightly awkward English constructions.  Also, I don't think it is necessary to mention interactive mode as this page is all about the **Dockerfile**.

-->https://gist.github.com/moxiegirl/c5b390b273f9fb732ccc

In some cases, you might start a process that itself creates more processes. If these are daemonized, non-child processes, you should be aware they can
cause unexpected behavior during `docker build`. 

A process you started within a Dockerfile might end before a daemonized
processes it created.  One solution is to check whether the daemonized process
is still running (for example, with `pgrep`) and `sleep` the parent process if it is.